docs/pages/includes/application-access/app-session.mdx

```diff
@@ -0,0 +1,35 @@
+Teleport records the HTTP activity within an Application Session. Only metadata
+such as path and status code from the HTTP activity is recorded, not the data
+sent or received by the application client.
+
+Within the Audit Log the `session_chunk_id` of the `App Session Data` record is
+the session id that activity is recorded against.  Once the `Session Uploaded`
+record is uploaded with that session id the activity is retrievable.
```

Comment:
What about instances with local log storage? I don't get a `Session Uploaded` record, does that mean I can not get the ID and/or retrieve the logs with `tsh`?